package.json

```diff
@@ -32,14 +32,17 @@
     "eslint-plugin-promise": "^4.0.1",
     "eslint-plugin-react": "^7.12.4",
     "eslint-plugin-standard": "^4.0.0",
+    "file-loader": "^3.0.1",
+    "svg-inline-loader": "^0.8.0",
+    "url-loader": "^1.1.2",
     "webpack": "^4.27.1",
     "webpack-cli": "^3.1.2"
   },
   "dependencies": {
-    "bootstrap": "^4.3.1",
     "prop-types": "^15.7.2",
     "react": "^16.6.3",
-    "react-bootstrap": "^1.0.0-beta.5",
-    "react-dom": "^16.6.3"
+    "react-dom": "^16.6.3",
+    "semantic-ui-css": "^2.4.1",
+    "semantic-ui-react": "^0.85.0"
```

Comment:
@siawo Why are we choosing SemanticUI-React (which is a React wrapper over the SemanticUI library) instead of a native React based library (e.g. Material-UI)?

Reply:
Semantic provides better predefined components which help in completing in the ui within the time frame.